docs/autopilot-knownissues.md

```diff
@@ -0,0 +1,17 @@
+# Autopilot - Known Issues
+
+## v1.24.0
+
+## Inefficient Autopilot Update for Controllers w/Worker Payloads
+
+[https://github.com/k0sproject/k0s/issues/1894](https://github.com/k0sproject/k0s/issues/1894)
+
+Autopilot as of `v1.24.0+k0s.0` can not cleanly shutdown the worker payload of controller node
```

Comment:
That version never existed. Not sure if we should either start with the first 1.24 release that really existed (or will exist, for that matter), or to just say "1.24". Same applies to line 3, of course.

Reply:
Good catch. I'll just use `1.24`